src/containers/PlanDetails/PlanDetails.component.js

```diff
@@ -12,6 +12,7 @@ import SwitchPlanPopup from 'components/SwitchPlanPopup';
 import CancelSwitchPopup from 'components/CancelSwitchPopup';
 import getSwitch from 'api/Customer/getSwitch';
 import { WrapStyled } from './PlanDetailsStyled';
+import GracePeriodError from '../../components/GracePeriodError/GracePeriodError';
```

Comment:
lets change to `import GracePeriodError from 'components/GracePeriodError';` and add a index file to GracePeriodError :)

Reply:
done ;)